copy.go

```diff
@@ -0,0 +1,46 @@
+package fssync
+
+import (
+	"io"
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+// copyContent is highly inspired from io.Copy, but calls to fadvise have been
+// added to prevent caching the whole content of the files during the process,
+// impacting the whole OS disk cache
+func (s *FsSyncer) copyContent(src, dst *os.File) (written int64, err error) {
+	buf := make([]byte, s.bufferSize)
+	for {
+		nr, er := src.Read(buf)
+		if nr > 0 {
+			if s.noCache {
+				unix.Fadvise(int(src.Fd()), written, written+int64(nr), unix.FADV_DONTNEED)
```

Comment:
As it's kinda low level, maybe add a comment to quickly explain what it does? And link to the man page (http://man7.org/linux/man-pages/man2/posix_fadvise.2.html)? because the Godoc of this method is empty: https://godoc.org/golang.org/x/sys/unix#Fadvise